src/components/ui/OverlinedText/OverlinedText.module.css

```diff
@@ -1,17 +1,41 @@
-.wordСontainer {
+.wordContainer {
   display: flex;
   flex-direction: row;
   align-items: center;
 }
 
 .line {
   display: block;
-  width: 91px;
+  width: 86px;
   height: 1px;
   background-color: var(--input-normal);
 }
 
 .word {
   padding: 0 12px;
-  font-size: 16px;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 1.5;
+  width: 115px;
+}
+
+@media screen and (min-width: 768px) {
+  .line {
+    width: 111px;
+  }
+
+  .word {
+    font-size: 14px;
+    width: unset;
+  }
+}
+
+@media screen and (min-width: 1440px) {
+  .line {
+    width: 104px;
+  }
+
```

Comment:
Can we make .line width dynamic?  Because if we change text between lines they can looks not like in design. And now they a little difference in login and signup